compiler/ballerina-lang/src/main/java/io/ballerina/projects/UsedTypeDefAnalyzer.java

```diff
@@ -0,0 +1,150 @@
+package io.ballerina.projects;
+
+import org.wso2.ballerinalang.compiler.PackageCache;
+import org.wso2.ballerinalang.compiler.bir.model.BIRNode;
+import org.wso2.ballerinalang.compiler.semantics.model.SimpleBTypeAnalyzer;
+import org.wso2.ballerinalang.compiler.semantics.model.types.BType;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+import org.wso2.ballerinalang.compiler.util.TypeTags;
+
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class UsedTypeDefAnalyzer extends SimpleBTypeAnalyzer<UsedTypeDefAnalyzer.AnalyzerData> {
```

Comment:
Missing Java doc comment

Reply:
Fixed in c812e4062d376b81a49f24f7fa16fc00a94b7783.